.jenkinsci/debug-build.groovy

```diff
@@ -40,7 +40,8 @@ def doDebugBuild(coverageEnabled=false) {
     + " -e IROHA_POSTGRES_USER=${env.IROHA_POSTGRES_USER}"
     + " -e IROHA_POSTGRES_PASSWORD=${env.IROHA_POSTGRES_PASSWORD}"
     + " --network=${env.IROHA_NETWORK}"
-    + " -v /var/jenkins/ccache:${CCACHE_DIR}") {
+    + " -v /var/jenkins/ccache:${CCACHE_DIR}"
+    + " -v /tmp/${GIT_COMMIT}-${BUILD_NUMBER}:/tmp/${GIT_COMMIT}") {
```

Comment:
Isn't `GIT_COMMIT` enough?

Reply:
We're using such construction to more easily correlate file system path with actual Jenkins job on a build machine